token.go

```diff
@@ -7,10 +7,11 @@ type TokenType string
 // TokenParams is the set of parameters that can be used when creating a token.
 // For more details see https://stripe.com/docs/api#create_card_token and https://stripe.com/docs/api#create_bank_account_token.
 type TokenParams struct {
-	Params   `form:"*"`
-	Bank     *BankAccountParams `form:"bank_account"`
-	Card     *CardParams        `form:"card"`
-	Customer string             `form:"customer"`
+	Params        `form:"*"`
+	Bank          *BankAccountParams `form:"bank_account"`
+	BankAccountId string             `form:"bank_account"`
```

Comment:
(I think the intention here was probably to remove `BankAccountId` as well as `CardId`.

Reply:
correct, missed that one